data/events.yml

```diff
@@ -0,0 +1,26 @@
+session001:
+  trial_bounds_file: events_Maxicosi_1_0months.csv
+  segment_number,surface,speed,seat,start_time,end_time
+  0,static,0,cot,1732789201649.05,1732789207245.48
+  1,Aula,5,cot,1732789098445.68,1732789119416.38
+  2,Aula,5,cot,1732789132630.74,1732789148623.54
+  3,Aula,5,cot,1732789181173.83,1732789192978.64
```

Comment:
I don't think these need to be yaml format. The CSV format is just fine. We can check those in as they already exist. The purpose of checking them in is because we manually edit them and it is best to track things you manually edit.